gpjax/linops/dense_linear_operator.py

```diff
@@ -107,7 +103,7 @@ def __mul__(self, other: float) -> LinearOperator:
 
         return DenseLinearOperator(matrix=self.matrix * other)
 
-    def _add_diagonal(self, other: DiagonalLinearOperator) -> LinearOperator:
+    def _add_diagonal(self, other: "gpjax.linops.diagonal_linear_operator.DiagonalLinearOperator") -> LinearOperator:
```

Comment:
Why do we need `gpjax.linops.diagonal_linear_operator` here? Can we not just have `DiagonalLinearOperator`?

Reply:
no, because it's in a different module, so you need to have the fully-qualified module name for beartype to be able to find the reference